### ISSUE TITLE
Kann keinen Pull Request erstellen

### ISSUE THREAD

**Issue by flori2212** [NoFear23m/LRPC_CommunityProject#7]
Da ich jetzt ja direkt an dem Projekt arbeite und wir nur einen Branch haben, kann ich keinen PullRequest erstellen.

Soll ich einen neunen Branch aufmachen? @NoFear23m 

-Florian

**Comment by NoFear23m** [comment]
Ja, gerne. Solange du weist wie du das dann falls es sein sollte in den MasterBranch rüberschiebst. Weil Git bin ich echt nicht so Fit.

**Comment by flori2212** [comment]
Oh, ich allerdings auch nicht. Soll ich dann lieber meine VM-Klassen in VB-Paradise posten
